src/app/App.js

```diff
@@ -70,15 +71,16 @@ export const App = ({ options }) => {
     withClasses: 'mainContainer__answersWrapper',
   });
 
-
-
-
   odpTrue.changeText('100% prawda');
 
   odpTrue.setSuccess();
   odpFalse.setDanger();
 
-
+  // Modal window
```

Comment:
Możesz znaleźć JSDoc i opisać w dokumentacji do funkcji ModalWindow przypadek użycia, albo po prostu odesłać do testów, bo one po to też są - pokazują nam jak coś stosować.

Reply:
o jaki JSDoc chodzi? Mam stworzyć jakiś plik osobn, do opisywania takich rzeczy?

Reply:
Dobra, nie w tym PR - pokaze Wam przy okazji. Możesz mergować pewnie. Masz już 3 approve nawet.